indexer/db/neon_tx_db.py

```diff
@@ -220,8 +220,22 @@ class _Record:
 
     logs: bytes | None
 
+    base_fee_per_gas: str
+    chain_id: int
+    priority_fee_per_gas: str
+
     @classmethod
     def from_tx(cls, neon_tx: NeonTxModel, neon_rcpt: NeonTxReceiptModel) -> Self:
+        base_fee_per_gas = None
+        priority_fee_per_gas = None
+        tx_chain_id = None
+        # Supported only for new transaction type
+        if neon_tx.tx_type == 2:
+            base_fee_per_gas = neon_tx.max_fee_per_gas - neon_tx.max_priority_fee_per_gas
+            priority_fee_per_gas = neon_tx.max_priority_fee_per_gas/base_fee_per_gas if base_fee_per_gas > 0 else 0
```

Comment:
IMHO, it is better to store the values as they were defined by an user.
And if it is required, convert them on the API level.